SignalUITests/SignalUITests.swift

```diff
@@ -0,0 +1,737 @@
+//
+//  SignalUITests.swift
+//  SignalUITests
+//
+//  Created by Matthew Kotila on 1/13/16.
+//  Copyright © 2016 Open Whisper Systems. All rights reserved.
+//
+
+import XCTest
+
+class SignalUITests: XCTestCase {
+    
+    override func setUp() {
+        super.setUp()
+        continueAfterFailure = false
+        XCUIApplication().launch()
+    }
+    
+    override func tearDown() {
+        super.tearDown()
+    }
+    
+    // requires unverified app
```

Comment:
Do you propose manually getting the app into this state and then running the tests? Or would you be able to get the app in this state as part of the test suite?